File: app/src/main/java/dev/ragnarok/fenrir/model/criteria/FaveProductsCriteria.java
package dev.ragnarok.fenrir.model.criteria;

import dev.ragnarok.fenrir.db.DatabaseIdRange;

public class FaveProductsCriteria extends Criteria {

    private final int accountId;

    private DatabaseIdRange range;

    public FaveProductsCriteria(int accountId) {
        this.accountId = accountId;
    }

    public DatabaseIdRange getRange() {
        return range;
    }

    public FaveProductsCriteria setRange(DatabaseIdRange range) {
        this.range = range;
        return this;
    }

    public int getAccountId() {
        return accountId;
    }
}

File: app_fenrir/src/main/kotlin/dev/ragnarok/fenrir/model/criteria/FaveProductsCriteria.kt
package dev.ragnarok.fenrir.model.criteria

import dev.ragnarok.fenrir.db.DatabaseIdRange

class FaveProductsCriteria(val accountId: Int) : Criteria() {
    var range: DatabaseIdRange? = null
        private set

    fun setRange(range: DatabaseIdRange?): FaveProductsCriteria {
        this.range = range
        return this
    }
}